### PR TITLE
Adopt ES6 in README

### PR DESCRIPTION
**Pull request by curran**
Noticed `var` in here and thought I'd propose this update.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ You could display a hexagon for each non-empty bin as follows:
 svg.selectAll("path")
   .data(hexbin(points))
   .enter().append("path")
-    .attr("d", function(d) { return "M" + d.x + "," + d.y + hexbin.hexagon(); });
+    .attr("d", (d) => "M" + d.x + "," + d.y + hexbin.hexagon());
```
Comment (Fil) [inline README.md]:
```suggestion
    .attr("d", d => `M${d.x},${d.y}${hexbin.hexagon()}`);
```

File: README.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ Alternatively, using a transform:
 svg.selectAll("path")
   .data(hexbin(points))
   .enter().append("path")
-    .attr("transform", function(d) { return "translate(" + d.x + "," + d.y + ")"; })
+    .attr("transform", (d) => "translate(" + d.x + "," + d.y + ")")
```
Comment (Fil) [inline README.md]:
```suggestion
    .attr("transform", d => `translate(${d.x},${d.y})`)
```

Comment (curran) [reply README.md]: Thanks @Fil !

**Comment by curran** [comment]
Accepted all suggestions.

**Pull request merged**